src/CMakeLists.txt

```diff
@@ -7,6 +7,10 @@ if (STORM_ENABLE_STEAM)
 endif()
 
 if (MSVC)
+    # Output to the same directory regardless of [CONFIG]
+    set(CMAKE_RUNTIME_OUTPUT_DIRECTORY_DEBUG "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}")
```

Comment:
I tested it on my PC and I don't have this issue with subdirectories inside `bin`.

Reply:
as discussed, this problem appears when generating sln rather than using CMakeLists directly